data/manager.rb

```diff
@@ -0,0 +1,74 @@
+require 'json'
+require 'pry'
+
+class Manager
+  RENTALS_PATH = './store/rentals.json'.freeze
+  BOOKS_PATH = './store/books.json'.freeze
+  PEOPLES_PATH = './store/peoples.json'.freeze
+
+  attr_accessor :books, :peoples, :rentals
+
+  def initialize(data = nil)
+    @data = data
+  end
+
+  def self.save_file(data)
+    new(data).save_file
+  end
+
+  def save_file
+    json_data.each do |item|
+      file_path = "./store/#{item.keys.first}.json"
+      File.write(file_path, item.values.first)
```

Comment:
- [ ] I noticed you've not made any changes from the last reviewer's suggestions. 
- On exit from your app, all data should be stored in the json files. It turns out there's a bug in this regard, as I got an error when exiting the application.

### View:
![image](https://github.com/PedroH-Neves/OOP-school/assets/50990925/04747e95-9099-4db9-b7c8-9ece509ee29c)

- You can review this [Article](https://ruby-doc.org/stdlib-3.0.0/libdoc/json/rdoc/JSON.html) for reference 👍